### DISCUSSION TITLE
Is Mitosis production ready?

### DISCUSSION THREAD

**Discussion by daniel-ign** [BuilderIO/jsx-lite#1173] [General]
Hello everyone! Im a developer and me and my team were looking into using Mitosis for an upcoming project of ours. Our main concern is that it may not be 100% production level ready, can anyone maybe provide some information on this? Would you recommend that we use it, or does it still need a little more time to fully evolve into a fully reliable project?

Either way though very impressive work and idea here by the Builder.io team, congratulations and we look forward to seeing future endeavours by you all!

**Comment by kingzez** [comment]
Official used in builder sdk, https://github.com/BuilderIO/builder/tree/main/packages/sdks.

And as a Mitosis user and contributor, I can tell you, Mitosis is not perfect, but it can help you solve most of the features and problems. I am currently using Mitosis to build my company's internal design system, and it may be open source in the near future.

**Comment by samijaber** [comment]
Yes, as @kingzez we use them for our SDKs which serve large Builder enterprise customers without issues.

Of course, it is still in beta and therefore bugs exist. But if you need a solution for building framework-agnostic components at work, I am confident that even if you need to spend time contributing to Mitosis (by raising issues or even fixing bugs yourself) to unblock yourself, there is simply no real competition or alternative today to what Mitosis does.

**Comment by ricovitch** [comment]
I would also be interested in a more detailed status : what's done, what's in progress, what's planned, what would need contribution... Anything that can help understand the status, the current limits, and potential risks of investing into mitosis.
Even if I'd be convinced myself, it would be useful to convince others.
The promise of this project looks huge to me, but I'm still concerned about how mature it is (not only technically, but also in terms of community and support).
In terms of competition, I would at least mention Haxe, which has been in the game since years. But maybe I'm missing something in terms of the lack of relevance of comparing it with mitosis.
